composer.json

```diff
@@ -174,13 +175,15 @@
     "phpunit/phpunit": "^9.3",
     "spiritix/php-chrome-html2pdf": "^1.6",
     "elasticsearch/elasticsearch": "^7.11",
-    "composer/composer": "*"
+    "composer/composer": "*",
+    "chrome-php/chrome": "^1.4.0"
```

Comment:
I'd make `ext-sockets` and `chrome-php/chrome` as `suggest` only - as this could possibly break an existing installation if `ext-sockets` is not enabled.